surefire-provider/src/test/java/org/arquillian/smart/testing/surefire/provider/LoaderVersionExtractorTestNgTest.java

```diff
@@ -6,14 +6,17 @@
 import java.net.URLClassLoader;
 import java.util.Arrays;
 import java.util.Collection;
+import net.jcip.annotations.NotThreadSafe;
 import org.jboss.shrinkwrap.resolver.api.maven.Maven;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(Parameterized.class)
+@Category(NotThreadSafe.class)
```

Comment:
Are we changed something here to make it `NotThreadSafe`?

Reply:
this failure: https://travis-ci.org/arquillian/smart-testing/builds/305775742#L2029
these tests are not-thread-safe